<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AllOptionDialog</class>
 <widget class="QDialog" name="AllOptionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>191</width>
    <height>82</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AllOptionDialog</string>
  </property>
  <widget class="QPushButton" name="pushButtonOK">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBoxAlpha">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>47</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Alpha:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCancel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>40</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
